### PR TITLE
[Feature] SMP-915: Integración del modulo de ubicaciones en el plugin de Cordova

### PR DESCRIPTION
**Pull request by Jaquilar**

### CODE REVIEW

**Review by ArianaQ1 — APPROVED** [review]

**Review by Smorales0492 — APPROVED** [review]

File: src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+         modules(
+            Koin.sentinelSDKModule("https://api-stage.sensys-iot.com")
+         ) 
+      }
+   }
+
+   override fun execute(
+         action: String,
+         args: JSONArray,
+         callbackContext: CallbackContext
```
Comment (brodriguezQXD) [inline src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt]: What about promises, can't we handle promises here?

File: src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+   ): Boolean {
+      if (action == "signIn") {
```
Comment (brodriguezQXD) [inline src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt]: If strategy seems pretty ugly, lets use when statement

File: src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt (hunk 1 above)
Comment (Jaquilar) [reply src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt]: execute method just override callbacks

Done

File: src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt (hunk 2 above)
Comment (Jaquilar) [reply src/android/org/apache/cordova/plugin/sentinel/CordovaPluginSentinelSDK.kt]: Done

**Review by brodriguezQXD — APPROVED** [review]

**Pull request merged**